cpp/models/ode_seair/README.md

```diff
@@ -0,0 +1,9 @@
+# ODE SEIR compartment model
+
+This model is a very simple ODE model with only four compartments and few parameters, mostly for demonstration of the MEmilio framework:
+- Susceptible, may become exposed at any time
```

Comment:
This is just copy paste from SEIR model and should be adapted!

Reply:
*done*